pallets/types/src/traits.rs

```diff
@@ -371,3 +371,42 @@ pub trait BridgeAssetLocker<AccountId> {
         amount: &Self::Balance,
     ) -> DispatchResult;
 }
+
+/// Temporary trait for Hashi bridge to handle asset lock and unlock
```

Comment:
If it is temporary and should be removed, please, add todo or link an issue. So, we don't forget it later.